caravel/assets/javascripts/SqlLab/components/SqlEditorLeft.jsx

```diff
@@ -102,22 +102,27 @@ class SqlEditorTopToolbar extends React.Component {
     const tableName = tableOpt.value;
     const qe = this.props.queryEditor;
     const url = `/caravel/table/${qe.dbId}/${tableName}/${qe.schema}/`;
+
+    this.setState({ tableLoading: true });
```

Comment:
ah you are right, sorry i missed that too. good catch!
@mistercrunch do you have a loading state you are adding to use with `this.state.tableLoading`?